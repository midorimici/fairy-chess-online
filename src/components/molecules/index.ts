export { BreadcrumbNav } from './BreadcrumbNav';
export { Container } from './Container';
export { GameList } from './GameList';
