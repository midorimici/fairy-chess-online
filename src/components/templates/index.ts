export { Game } from './Game';
export { Games } from './Games';
export { Home } from './Home';
export { NewRoom } from './NewRoom';
