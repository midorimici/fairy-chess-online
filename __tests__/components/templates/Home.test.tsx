import { render, screen } from '@testing-library/react';
import { Home } from 'components/templates';

describe('Home', () => {
  it('renders unchanged', () => {
    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
  });

  it('contains copy right', () => {
    render(<Home />);
    const copyRightElement = screen.getByText(/Â©/);
    expect(copyRightElement).toBeInTheDocument();
  });
});
