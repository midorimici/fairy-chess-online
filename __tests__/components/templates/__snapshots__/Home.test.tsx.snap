// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders unchanged 1`] = `
<DocumentFragment>
  <div
    class="chakra-container css-0"
  >
    <main
      class="css-0"
    >
      <a
        class="chakra-link css-0"
        href="/games"
      >
        games
      </a>
    </main>
    <div
      class="css-0"
    >
      Â© 2022-22 midorimici
    </div>
  </div>
</DocumentFragment>
`;
